src/kv/value.rs

```diff
@@ -519,6 +548,16 @@ pub trait VisitValue<'v> {
         self.visit_str(&*value.encode_utf8(&mut b))
     }
 
+    /// Visit an IP Address
+    fn visit_ip_addr(&mut self, value: IpAddr) -> Result<(), Error> {
```

Comment:
I don't think we should expose visitors for these types, and instead let them fall through to `visit_any`.